index.js

```diff
@@ -0,0 +1,97 @@
+import { catsData } from "/data.js"
+
+const catsArray = []
+const emotionsArray = []
+const emotionRadios = document.getElementById('emotion-radios')
+const radios = document.getElementsByClassName('radio')
+const getImageBtn = document.getElementById('get-image-btn')
+const gifsOnlyCheckbox = document.getElementById('gifs-only-option')
+const memeModalInner = document.getElementById('meme-modal-inner')
+const memeModal = document.getElementById('meme-modal')
+const memeModalClose = document.getElementById('meme-modal-close-btn')
+
+emotionRadios.addEventListener('change', highlightCheckedOption)
+getImageBtn.addEventListener("click", renderCat)
+memeModalClose.addEventListener("click", closeModal)
+
+function closeModal() {
+    memeModal.style.display = "none"
+}
+
+function getMatchingCatsArray() {
+    if (document.querySelector('input[type="radio"]:checked')) {
```

Comment:
I'd put this in in a variable so you don't have to query it multiple times:

```js
const checkedElement = document.querySelector('input[type="radio"]:checked')

if (checkedElement) {
  const selectedEmotion = checkedElement.value
```

Reply:
Addressed, implemented as recommended.